dev/console/console.js

```diff
@@ -169,7 +169,7 @@ var CKCONSOLE = ( function() {
 					valueName;
 
 				for ( valueName in values )
-					this.valuesElements[ valueName ].setHtml( values[ valueName ] );
+					this.valuesElements[ valueName ].setHtml( CKEDITOR.tools.htmlSafeByReview(values[ valueName ], 'The values are being pulled from the editor using definition.refresh') );
```

Comment:
What does `this.definition.refresh` do? Can we trust the output?

Reply:
`this.definition.refresh` is passed in from the `panelDefinition` variable which would include some configs that would be defined by the developer. The usages I see internally either don't involve dangerous values or are sanitized.